File: shiva_sakkthi_printers/shiva_sakkthi_printers/erp_backup/bom.py
Core - Line Number:1133+, 1134+
  Version - 13.19.0
  Core Path: erpnext/manufacturing/doctype/bom/bom.py
   Existing Code:
    ```
   
    
    ```
   Code to be changed:
    ```
    	if filters and filters.get('filter_by_item_group'):
		  query_filters['item_group'] = 'By-Product'
    
    ```

File: shiva_sakkthi_printers/shiva_sakkthi_printers/erp_backup/bom.js
Core - Line Number:51 to 52
  Version - 13.19.0
  Core Path: erpnext/manufacturing/doctype/bom/bom.js
   Existing Code:
    ```
    "item_code": doc.item
    
    ```
   Code to be changed:
    ```
    	"item_code": doc.item,
			"filter_by_item_group": 1
    
    ```
